### PR TITLE
TP-5

### PR DESCRIPTION
**Pull request by Brandon-B9845**
Added in feature so that the default position is the users geolocation. Complete with comments!

### CODE REVIEW

File: hooks/FindLocation.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react"
+
+const FindLocation = async () => {
+    interface Location {
+        latitude: number,
+        longitude: number
+    }
+
+    // const [cords, setCords] = useState({lattitude: 0, longitude: 0})
```
Comment (CSpiegelhalter) [inline hooks/FindLocation.tsx:9]: stray comment

File: hooks/FindLocation.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        } else {
+            reject("your browser doesn't support geolocation API")
+        }
+    })
+
+
+   
+    const center: any = async () => {
+        // Now I can use the promise followed by .then() 
+        // to make use of the values anywhere in the program
```
Comment (CSpiegelhalter) [inline hooks/FindLocation.tsx:39]: This comment doesn't seem like it belongs here?

File: hooks/FindLocation.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+}
+
+export default FindLocation
+
+
+// use state like this ?????
+// useEffect(() => {
+//     fetch(info).then(setState(info))
+// }, [])
```
Comment (CSpiegelhalter) [inline hooks/FindLocation.tsx:60]: stray comment

File: pages/index.tsx [hunk 4]
```diff
@@ -3,16 +3,33 @@ import Image from 'next/image'
 import { Inter } from 'next/font/google'
 import styles from '@/styles/Home.module.css'
 import NavBar from '@/components/NavBar'
-import { use, useMemo } from 'react'
+import { use, useEffect, useMemo } from 'react'
```
Comment (CSpiegelhalter) [inline pages/index.tsx:6]: [qq] Are you using useMemo?

**Review by CSpiegelhalter — APPROVED** [review]
Remove comments

File: pages/index.tsx (hunk 4 above)
Comment (Brandon-B9845) [reply pages/index.tsx:6]: Yes line 48 of index.tsx

**Pull request merged**